mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -2130,12 +2144,12 @@ struct GridwiseGemmRewritePattern : public OpRewritePattern<miopen::GridwiseGemm
 
     // Emit BlockwiseCopy ops.
     auto blockwiseCopyA = b.create<miopen::BlockwiseCopyOp>(
-        loc, op.filter(), lds2DMatrixASubviewOp, blockwiseCopyASrc,
-        blockwiseCopyADst, threadAAllocOp);
+        loc, op.filter(), lds2DMatrixASubviewOp, blockwiseCopyASrcVector,
+        blockwiseCopyADstVector, threadAAllocOp);
```

Comment:
Is vector load length (=1) specified somewhere?

Reply:
how much data is load is not specified from the syntax of `blockwise_copy` op. It's an implementation detail inside `threadwise_copy`.